xbmc/filesystem/FileCache.cpp

```diff
@@ -116,12 +116,12 @@ bool CFileCache::Open(const CURL& url)
 
   m_sourcePath = url.GetRedacted();
 
-  CLog::Log(LOGDEBUG,"{} - <{}> opening", __FUNCTION__, m_sourcePath);
+  CLog::Log(LOGDEBUG, "CFileCache::Open - <{}> opening", m_sourcePath);
```

Comment:
```suggestion
  CLog::Log(LOGDEBUG, "CFileCache::{} - <{}> opening", __FUNCTION__, m_sourcePath);
```

Should work fine everywhere. We use this syntax in many places.

Reply:
Good idea: Updated :-)